client/templates/initiative/create/initiativeCreate.js

```diff
@@ -2,6 +2,13 @@ Template.initiativeCreate.onRendered(function() {
   $('select').material_select();
 });
 
+Template.initiativeCreate.helpers({
+  overLimit: function(){
+    var initiatives = Initiatives.find({createdBy: Meteor.userId()}).count();
```

Comment:
Can you test this on a user with no Initiatives? Will it crash the client (calling .count() on an undefined object if mongo returns undefined for no matches?)